assets/js/base/context/cart-checkout/checkout-state/actions.js

```diff
@@ -5,8 +5,11 @@ import { TYPES } from './constants';
 
 const {
```

Comment:
Can we get inline docs here matching your PR description?

Reply:
This set of things is all the dispatch actions constants. If you're referring to the status descriptions from the pull description, this is documented with the `CheckoutStatusConstants` type-def in `@woocommerce/type-defs/checkout.js`.  Also in the same type-def file is the `CheckoutDispatchActions` type-def which documents the _publicly exposed_ dispatch actions (publicly disposed actions on the context do not include `setPristine` so I can't document `@type {CheckoutDispatchActions}` as an inline doc here.  All the documentation exists for the publicly exposed api via the context itself (this file is internal to the context only).